### ISSUE TITLE
Fix accessibility issues in tab ordering

### ISSUE THREAD

**Issue by Mattsi-Jansky** [Mattsi-Jansky/matt.si#811]
Tab ordering is important to screen reader users. Right now, the tab order on the main page stats on the left and goes through all the errata links before reaching the content. When it does reach the content it highlights two links for every article. 

Each image link to each article should be skipped in the tab order such that each article only comes up once, and the errata in the left pane should come up second not first in the tab order. There will likely be other issues once I look closer too.
